lib/settings/about_screen.dart

```diff
@@ -39,6 +39,20 @@ Future<String> executeCommand(String command, List<String> arguments) async {
 }
 
 Future<String> getRaspberryPiModel() async {
```

Comment:
Should we rename this to `getDeviceModel` or something?

Reply:
Agree. Resolved by [6fe2d43](https://github.com/TheContrappostoShop/Orion/pull/18/commits/6fe2d43bcfb362630f12385318261b280d0e01be)